src/Kunstmaan/SeoBundle/Resources/config/routing.yml

```diff
@@ -0,0 +1,3 @@
+Seo_Menuadaptor:
+    resource: "@KunstmaanSeoBundle/Controller/Admin/SettingsController.php"
+    type:     annotation
```

Comment:
The robotscontroller needs to be added in app/config because the routing of the SEO bundle will be prefixed with /admin and the robot.txt file lives in document root.
?